WD-Firmware/psp/psp_specific/api_inc/psp_specific_definitions.h

```diff
@@ -0,0 +1,70 @@
+/* 
+* SPDX-License-Identifier: Apache-2.0
```

Comment:
change name to core name and have few files as on the onenote

Reply:
file removed